acceptance/config_test.go

```diff
@@ -65,58 +62,53 @@ type ServerStub struct {
 	}
 }
 
-// FindConfig finds the closest config file.
-func FindConfig(t *testing.T, dir string) (string, bool) {
-	shared := false
+// FindConfigs finds the closest config file.
+func FindConfigs(t *testing.T, dir string) []string {
+	configs := []string{}
 	for {
 		path := filepath.Join(dir, configFilename)
 		_, err := os.Stat(path)
 
 		if err == nil {
-			return path, shared
+			configs = append(configs, path)
 		}
 
-		shared = true
-
 		if dir == "" || dir == "." {
 			break
 		}
 
-		if os.IsNotExist(err) {
-			dir = filepath.Dir(dir)
+		dir = filepath.Dir(dir)
```

Comment:
Is the path guaranteed to never be absolute?

If it is, this loop will be infinite.

Reply:
Yes, in this context is always relative. If it was absolute, the logic would not work anyway as we don't want to search for configs outside of acceptance/.

Added a comment about this requirement.